src/Utilities/Requires.hpp

```diff
@@ -11,7 +11,7 @@
 namespace Requires_detail {
 template <bool B>
 struct requires_impl {
-  using template_error_types_failed_to_meet_requirements_on_template_parameters =
+  using template_error_types_failed_to_meet_requirements_on_template_parameters=
```

Comment:
We don't require clang-format and it shouldn't be applied to the entire repository. If we have the `off` and `on` everywhere it will turn into a huge mess. If it was only here, I'd agree.